specification.markdown

```diff
@@ -579,9 +579,9 @@ The maxver attribute is OPTIONAL. If the maxver attribute is not present then th
 
 ### \<documentation> Element
 
-Within an EBML Schema, the XPath of `<documentation>` attribute is `/EBMLSchema/element/documentation`.
+Within an EBML Schema, the XPath of `<documentation>` element are `/EBMLSchema/element/documentation` and `/EBMLSchema/element/restriction/enum/documentation`.
 
-The `<documentation>` element provides additional information about the EBML Element. Within the `<documentation>` element the following XHTML [@!W3C.SPSD-xhtml-basic-20180327] elements MAY be used: `<a>`, `<br>`, `<strong>`.
+The `<documentation>` element provides additional information about EBML Elements or enumeration values. Within the `<documentation>` element the following XHTML [@!W3C.SPSD-xhtml-basic-20180327] elements MAY be used: `<a>`, `<br>`, `<strong>`.
```

Comment:
Is this still valid? If I remember carefully, _XHTML Basic_ has been superseded by W3C two years ago or so.

Reply:
The [xsd](https://github.com/cellar-wg/ebml-specification/blob/master/EBMLSchema.xsd) is valid, the commit resolves the discrepancy between the narrative and the xsd. The reference to XHTML Basic is unchanged by the commit and not in scope of commit, so I suggest it is not blocking, but feel welcome to file an issue.

Reply:
It’s not blocking. I just noticed it while reading this PR. In the meantime I checked, XHTML Basic has been superseded on 27 March 2018.

Reply:
>The reference to XHTML Basic is unchanged by the commit and not in scope of commit, so I suggest it is not blocking, but feel welcome to file an issue.

done in https://github.com/cellar-wg/ebml-specification/issues/341